app/views/users/edit.html.erb

```diff
@@ -4,6 +4,14 @@
 
   <%= render 'form', user: @user %>
 
-  <%= link_to 'Show', @user %> |
-  <%= link_to 'Back', users_path %>
+  <%= link_to 'Delete Account', @user, method: :delete,
+                                       class: "btn btn-danger",
+                                       data: { confirm: "Are you sure? There's no going back!" } %>
```

Comment:
Could we explicitly say that they're about to delete their account? Sometimes I hit a button without realizing what I'm doing.